samples/ko/Getting Started/index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html>
+    
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>BRACKETS으로 시작하기</title>
+        <meta name="description" content="An interactive getting started guide for Brackets.">
+        <link rel="stylesheet" href="main.css">
+    </head>
+    <body>
+        
+        <h1>BRACKETS으로 시작하기</h1>
+        <h2>이것은 당신의 가이드입니다.</h2>
+        
+        <!--
+            MADE WITH <3 AND JAVASCRIPT
+        -->
+        
+        <p>
+            
+            웹 디자인을 이해하는 현대적인 오픈 소스 코드 편집기 인 Brackets에 오신 것을 환영합니다. 가볍고 강력한 코드 편집기로 시각 도구를 편집기에 통합하여 원할 때 도움을받을 수 있습니다.
+        </p>
+        
+        <!--
+            WHAT IS BRACKETS?
+        -->
+        <p>
+            <em>Brackets은 다른 유형의 편집기입니다.</em>
+            Brackets 에는 다른 편집자에서 볼 수없는 빠른 편집, 실시간 미리보기 및 기타 기능과 같은 고유 한 기능이 있습니다. Brackets에는 JavaScript, HTML 및 CSS로 작성됩니다. 즉, 대다수 사용자는 대괄호를 사용하여 편집자를 수정하고 확장하는 데 필요한 기술을 보유하고 있습니다. 사실, 우리는 Brackets을 만들기 위해 Brackets을 매일 사용합니다. 주요 기능 사용 방법에 대한 자세한 내용은 다음을 참조하십시오.
+        </p>
+        
+        <!--
+            GET STARTED WITH YOUR OWN FILES
+        -->
+        
+        <h3>Brackets 안에 있는 프로젝트</h3>
+        <p>
+           Brackets을 사용하여 자신의 코드를 편집하려면 파일이 들어있는 폴더를 열면 됩니다.  Brackets은 현재 열려있는 폴더를 "프로젝트"로 취급합니다. 코드 힌트, 실시간 미리보기 및 빠른 편집과 같은 기능은 현재 열려있는 폴더 내의 파일에만 사용합니다.
+        </p>
+        
+        <samp>
+            이 샘플 프로젝트에서 벗어나 자신의 코드를 편집 할 준비가되면 왼쪽 사이드 바의 드롭 다운을 사용하여 폴더를 전환 할 수 있습니다. 지금은 드롭 다운에 "시작하기"라고 표시되어 있습니다. 바로 지금보고있는 파일이 들어있는 폴더입니다. 드롭 다운을 클릭하고 "폴더 열기 ..."를 선택하여 자신의 폴더를 엽니 다. 나중에이 드롭 다운을 사용하여이 샘플 프로젝트를 포함하여 이전에 열어 본 폴더로 다시 전환 할 수도 있습니다.
+        </samp>
+        
+        <!--
+            THE RELATIONSHIP BETWEEN HTML, CSS AND JAVASCRIPT
+        -->
+        <h3>CSS 및 JavaScript 용 빠른 편집</h3>
+        <p>
+            문서를 더 이상 전환하지 않고 컨텍스트를 잃어 버리지 않습니다. HTML을 편집 할 때 <kbd> Cmd / Ctrl + E </ kbd> 단축키를 사용하여 모든 관련 CSS를 표시하는 빠른 인라인 편집기를 엽니다.
```

Comment:
Here and in the other places you should close the html tags correctly: no whitespace after `</`